app/javascript/techradar.js

```diff
@@ -55,76 +63,131 @@
+                    .attr("ry", 5)
+                    .attr("height", 25)
+                    .attr("stroke-width", 3)
+                    //Following will be set when showing
+                    .attr("width", 0)
+                    .attr("x", 0)
+                    .attr("y", 0);
+
+                tooltipGroup.append("text")
+                    .attr("class", "spotTooltip");
+            }
+
             function renderHelp(helpCanvas, radar) {
                 var animationSpeed = 200,
                     iconRadius = 13,
                     xPosition = 18, yPosition = 18;
-                
+
                 function lowlightHelp() {
                     helpCanvas.selectAll('text.helpLabel').remove();
                     d3.select('.helpIcon').transition().ease('cubic-out').duration(animationSpeed).attr("r", iconRadius);
                 }
-                
+
                 function highlightHelp() {
-                    helpCanvas.append('text').attr({'class':'helpLabel', 'dx': xPosition * 2, 'dy': xPosition + 5}).text('Help');
+                    helpCanvas.append('text').attr({
+                        'class': 'helpLabel',
+                        'dx': xPosition * 2,
+                        'dy': xPosition + 5
+                    }).text('Help');
                     d3.select('.helpIcon').transition().ease('cubic-out').duration(animationSpeed).attr("r", iconRadius + 2);
                 }
+
                 function toggleHelp() {
                     $('#help').modal({
-                        opacity:60, autoResize: true, overlayClose: true, overlayCss: {backgroundColor:"#fff" },
+                        opacity: 60, autoResize: true, overlayClose: true, overlayCss: {backgroundColor: "#fff"},
                         onOpen: function (dialog) {
                             dialog.overlay.fadeIn();
                             dialog.data.fadeIn();
                             dialog.container.fadeIn();
-                }   }); }
+                        }
+                    });
+                }
 
                 // render the help-text
                 var helpDiv = d3.select('#help').append('div');
                 helpDiv.append('h2').html(radar.title);
                 helpDiv.append('div').html(radar.description);
-                _.forEach(radar.arcs, function(arc) {
+                _.forEach(radar.arcs, function (arc) {
                     helpDiv.append('h3').html(arc.title);
                     helpDiv.append('div').html(arc.description);
                 });
-                
+
                 helpCanvas.append("circle")
                     .attr({'class': 'helpIcon', "cx": xPosition, "cy": yPosition, "r": iconRadius})
-                    .on('mouseout', function () { lowlightHelp(); })
-                    .on('mouseover', function () { highlightHelp(); })
-                    .on('click', function() { toggleHelp(); });
+                    .on('mouseout', function () {
```

Comment:
Can't help - have to mention it at least once - the reformatting is more than counterproductive - have a hard time to see the effective changes.